lib/axon.ex

```diff
@@ -1136,10 +1127,12 @@ defmodule Axon do
     end
   end
 
+  defp unique_id(modifiers) when is_list(modifiers), do: System.unique_integer(modifiers)
```

Comment:
I don't think this is really all that necessary because it's an extra call to do the same thing without avoiding any duplication

Reply:
Thanks, I'll try and add something with more value next time.